src/components/index.d.ts

```diff
@@ -11,6 +11,15 @@ declare class UniversalNavbar extends React.Component {
 }
 export { UniversalNavbar }
 
+declare class UniversalNavbarExpanded extends React.Component {
```

Comment:
We should make a type test index.tests

Reply:
Victor started on some work for this file in #61